.eslintrc.json

```diff
@@ -212,8 +212,8 @@
 				"restrictions": [
 					"vs/nls",
 					"azdata",
-					"mssql",
 					"azurecore",
+					"mssql",
```

Comment:
Please put these back where they were - no reason for these to change in this PR